handler.ts

```diff
@@ -52,6 +57,27 @@ Bugsnag.start({
 // For reference, see https://docs.bugsnag.com/platforms/javascript/aws-lambda/#usage
 const bugsnagHandler = Bugsnag.getPlugin('awsLambda').createHandler()
 
+const getPrimaryGeocoder = () => {
```

Comment:
This can probably now be refactored to just pass `GEOCODER` right into `type`

Reply:
Hm, yeah, I thought that too, but then I thought that since the arguments were slightly different they should stay separate. But, I suppose passing in baseUrl even if it's undefined is totally fine! Thanks for the feedback.